### PR TITLE
Add new contributing page

### PR DESCRIPTION
**Pull request by Chanel-Diep**
Add new contributing page so readers know how to make pull requests for the manual.

### CODE REVIEW

File: src/contributing/contributing.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
```
Comment (yndajas) [inline src/contributing/contributing.md:1]: You'll want this to be `src/contributing.md` or `src/contributing/index.md`

File: src/contributing/contributing.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+title: Contributing to the dxw Accessibility Manual
+navigation_order: 0
+---
+
+The dxw Accessibility Manual is public and anyone, inside or outside of dxw, can suggest changes to it. This means you!
+
+We manage changes via GitHub ‘Pull Requests’. This is a technical term for requesting a specific change is made to a repository of text. Often this text is code, but in this case it’s the contents of this Playbook. It allows everyone to see exactly what changes are being proposed, and for discussion and changes to be made before it is ‘merged’, and those changes to be recorded for the future.
```
Comment (yndajas) [inline src/contributing/contributing.md:8]: `Playbook` -> `accessibility manual`?

File: src/contributing/contributing.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+If you’re experienced in doing this, then all you need to know is that this repository is available at https://github.com/dxw/playbook
```
Comment (yndajas) [inline src/contributing/contributing.md:10]: `https://github.com/dxw/accessibility-manual`

File: src/contributing/contributing.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+If creating a pull request is new to you, then don’t worry, here is a guide to use Netlify CMS to create one for you.
+
+## What you will need
+
+You will need a GitHub account. If you don’t already have one, then sign up here. You do not need to be added to the dxw GitHub organisation to be able to suggest changes.
```
Comment (yndajas) [inline src/contributing/contributing.md:16]: `here` - missing link?

File: src/contributing/contributing.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+navigation_order: 0
+---
+
+The dxw Accessibility Manual is public and anyone, inside or outside of dxw, can suggest changes to it. This means you!
+
+We manage changes via GitHub ‘Pull Requests’. This is a technical term for requesting a specific change is made to a repository of text. Often this text is code, but in this case it’s the contents of this Playbook. It allows everyone to see exactly what changes are being proposed, and for discussion and changes to be made before it is ‘merged’, and those changes to be recorded for the future.
+
+If you’re experienced in doing this, then all you need to know is that this repository is available at https://github.com/dxw/playbook
+
+If creating a pull request is new to you, then don’t worry, here is a guide to use Netlify CMS to create one for you.
```
Comment (yndajas) [inline src/contributing/contributing.md:12]: `Netlify CMS` is now `Decap CMS`, but it doesn't really matter!

File: src/contributing/contributing.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+For an existing page:
+
+Navigate to the page you want to edit
+Click on the yellow ‘Edit this page’ link
+For an new page:
+
+Navigate to the most relevant ‘parent’ for the new page
+Click on the yellow ‘Add a new page here’ link
+Alternatively, if you prefer to use the admin backend directly:
```
Comment (yndajas) [inline src/contributing/contributing.md:30]: These two blocks become two paragraphs. You'll want some list or number list markup here and in some other places I think

You can preview what this looks like here: https://deploy-preview-133--startling-pasca-a7e8b4.netlify.app/contributing/contributing

File: src/contributing/contributing.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+3. Select ‘fork the repo’ if you are asked whether you want to fork the repo
+
+4. Make your edits in the ‘Rich Text’ editor
+
+5. Click ‘Save’ at the top of the page
+
+6. Click ‘Status: Draft’ and choose ‘In review’
+
+Your suggested changes will not be live yet. Your suggestions should be available to view on the GitHub Pull Requests page. After a few minutes, you should be able to view a preview of the Playbook with your changes by clicking on the URL next to ‘😎 Browse the preview’
```
Comment (yndajas) [inline src/contributing/contributing.md:47]: `Playbook` -> `Accessibility Manual`

**Review by yndajas — COMMENTED** [review]
Generally looks good, but a few small things to fix

I wonder if here and in the playbook version we should talk more about the CMS up front, then talk about the pull request process later - I feel like we might scare people off by saying you need to do pull requests before saying there's a handy CMS to manage most of the content editor process for you 😅

File: src/contributing/contributing.md (hunk 6 above)
Comment (Chanel-Diep) [reply src/contributing/contributing.md:30]: Thank you, I got stuck with the double sets of numbering in the Playbook so will rewrite this section

File: src/contributing/index.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+2. Log in with your GitHub account
+
+3. Select ‘fork the repo’ if you are asked whether you want to fork the repo
+
+4. Make your edits in the ‘Rich Text’ editor
+
+5. Select ‘Save’ at the top of the page
+
+6. Select ‘Status: Draft’ and choose ‘In review’
```
Comment (yndajas) [inline src/contributing/index.md:32]: Should these have full stops like the others? (I think generally there's a bit of inconsistency on full stops 😱)

File: src/contributing/index.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+## Reviewing changes
+
+After a few minutes, you should be able to view a preview of the Accessibility Manual with your changes by clicking on the URL next to ‘😎 Browse the preview'. Your suggested changes will not be live yet. Your suggestions should be available to view on the GitHub Pull Requests page.
+
+Everyone’s suggested changes that are being reviewed are publicly available. Anyone can publicly comment on a suggested change. Only GitHub accounts that have been added to the dxw GitHub organisation and staff team can ‘merge’ a pull request and make it live. If you are not on this list and want to be, please email support@dxw.com with your GitHub username.
+
+To see the suggested changes, you can click the ‘Files changed’ tab to see lines of text that have been removed (in red) and lines of text that have been added (in green). Alternatively you can click on the URL next to ‘😎 Browse the preview’ to see what the Playbook will look like if this change is accepted.
```
Comment (yndajas) [inline src/contributing/index.md:58]: `Playbook` -> `Accessibility Manual`

**Review by yndajas — APPROVED** [review]
Looking really good! There's one very minor thing about full stops that I don't really care about, but could be fixed. Otherwise just one remaining playbook reference to fix and it'll be good to go

File: src/contributing/index.md (hunk 9 above)
Comment (chaneldiepmoj) [reply src/contributing/index.md:58]: argh thank you!

File: src/contributing/index.md (hunk 8 above)
Comment (Chanel-Diep) [reply src/contributing/index.md:32]: I can come back to this but thank you for pointing it out :)

**Pull request merged**